asv_bench/benchmarks/benchmarks.py

```diff
@@ -819,4 +821,147 @@ def time_index(self, shape):
         return self.df.index
 
 
+class TimeIndexingNumericSeries:
+
+    param_names = ["shape", "dtype", "index_structure"]
+    params = [
+        get_benchmark_shapes("TimeIndexingNumericSeries"),
+        (np.int64, np.uint64, np.float64),
+        ("unique_monotonic_inc", "nonunique_monotonic_inc"),
+    ]
+
+    def setup(self, shape, dtype, index_structure):
+        N = shape[0]
+        indices = {
+            "unique_monotonic_inc": Index(range(N), dtype=dtype),
+            "nonunique_monotonic_inc": Index(
+                list(range(55)) + [54] + list(range(55, N - 1)), dtype=dtype
+            ),
+        }
+        self.data = Series(np.random.rand(N), index=indices[index_structure])
+        self.array = np.arange(800)
+        self.array_list = self.array.tolist()
+
+    def time_getitem_scalar(self, shape, index, index_structure):
+        self.data[800]
+
+    def time_getitem_slice(self, shape, index, index_structure):
+        execute(self.data[:800])
+
+    def time_getitem_list_like(self, shape, index, index_structure):
+        execute(self.data[[800]])
+
+    def time_getitem_array(self, shape, index, index_structure):
+        execute(self.data[self.array])
+
+    def time_getitem_lists(self, shape, index, index_structure):
+        execute(self.data[self.array_list])
+
+    def time_iloc_array(self, shape, index, index_structure):
+        execute(self.data.iloc[self.array])
+
+    def time_iloc_list_like(self, shape, index, index_structure):
+        execute(self.data.iloc[[800]])
+
+    def time_iloc_scalar(self, shape, index, index_structure):
+        self.data.iloc[800]
+
+    def time_iloc_slice(self, shape, index, index_structure):
+        execute(self.data.iloc[:800])
+
+    def time_loc_array(self, shape, index, index_structure):
+        execute(self.data.loc[self.array])
+
+    def time_loc_list_like(self, shape, index, index_structure):
+        execute(self.data.loc[[800]])
+
+    def time_loc_scalar(self, shape, index, index_structure):
+        self.data.loc[800]
+
+    def time_loc_slice(self, shape, index, index_structure):
+        execute(self.data.loc[:800])
+
+
+class TimeReindex:
+    param_names = ["shape"]
+    params = [get_benchmark_shapes("TimeReindex")]
+
+    def setup(self, shape):
+        Rows, Cols = shape
+        rng = IMPL.date_range(start="1/1/1970", periods=Rows, freq="1min")
+        self.df = DataFrame(np.random.rand(Rows, Cols), index=rng, columns=range(Cols))
+        self.df["foo"] = "bar"
+        self.rng_subset = Index(rng[::2])
+        self.df2 = DataFrame(
+            index=range(Rows), data=np.random.rand(Rows, Cols), columns=range(Cols)
+        )
+        N = Rows
+        K = Cols
```

Comment:
Could we just use `rows` and `cols` in all places?

Reply:
Changed to rows and cols